spec/ConfigChanges/ConfigFile.spec.js

```diff
@@ -116,6 +121,21 @@ describe('ConfigFile tests', function () {
 
                 expect(ConfigFile.resolveConfigFilePath('', 'ios', '*-Info.plist')).toBe(expectedPlistPath);
             });
+
+            it('should return Info.plist file', function () {
+                const projName = 'XXX';
+                const expectedPlistPath = path.join(projName, 'Info.plist');
+
+                ConfigFile.__set__('getIOSProjectname', () => projName);
```

Comment:
The jasmine equivalent should be:
```suggestion
                spyOn(ConfigFile, 'getIOSProjectname').and.returnValue(projName);
```